db/pre-re/mob_db.yml

```diff
@@ -37483,7 +37483,7 @@ Body:
     AttackMotion: 768
     ClientAttackMotion: 0
     DamageMotion: 576
-    Class: Guardian
+    Class: Boss
```

Comment:
For consistency and future reference better move this to another PR

Reply:
Ok, but it's required for Flasher to ignore these.

Reply:
DB changes are now here: https://github.com/rathena/rathena/pull/9359